user/apis/serializers.py

```diff
@@ -0,0 +1,15 @@
+from rest_framework import serializers
+
+from user.models import User
+
+
+class UserSerializer(serializers.ModelSerializer):
+    team_members = serializers.SerializerMethodField()
```

Comment:
use a nested serializer instead, which would contain the name of the user and id or the user

for that figure out how to write nested serializer